Phoenix/PHShebangPreprocessor.m

```diff
@@ -74,10 +74,24 @@ + (NSString *) process:(NSString *)script atPath:(NSString *)path error:(NSError
         return script;
     }
 
-    // Read past shebang-directive
-    [standardOutputFile readDataOfLength:scanner.scanLocation];
+    // Read past shebang-directive (hold onto it incase it's not the shebang text)
```

Comment:
I’m wondering if it might be easier and safer to create a new scanner for the output. Check if the command is present with `readCommand:`, if it is, the scanner will be moved forward, if not it, will stay intact. Then just return the output string based on the position of the scanner. (I’m thinking of an edge case where the preprocessor might even reformat the shebang.)